README.md

```diff
@@ -190,18 +190,18 @@ Date:   Fri Jan 25 19:25:23 2019 +0100
       See the tutorial_wip branch for the actual commit history
 ```
 
-In there we see a few interesting things: 
+Here we see a few interesting things: 
 * The first two commits are made by me.
 * Your initial commit to add Bob is the current _HEAD_ of the _master_ branch on the _Remote Repository_. We'll look at this again when we talk about branches and getting remote changes.
 * The latest commit in the _Local Repository_ is the one we just made, and now we know its hash.
 
-> Note that the actual commit hashes will be different for you. If you want to know how exactly git arrives at those revision IDs have a look at [this interesting article ](https://blog.thoughtram.io/git/2014/11/18/the-anatomy-of-a-git-commit.html).
+> Note that the actual commit hashes will be different for you. If you want to know how exactly git arrives at those revision IDs have a look at [this interesting article](https://blog.thoughtram.io/git/2014/11/18/the-anatomy-of-a-git-commit.html).
 
-To compare that commit and the one one before we can do `git diff <commit>^!`, where the `^!` tells git to compare to the commit one before. So in this case I run `git diff 87a4ad48d55e5280aa608cd79e8bce5e13f318dc^!`
+To compare that commit and a previous one we can do `git diff <commit>^!`, where the `^!` tells git to compare to the previous commit. So in this case I run `git diff 87a4ad48d55e5280aa608cd79e8bce5e13f318dc^!`.
```

Comment:
Hm.. that's specifically _the_ previous commit, not _a_ previous commit. But having that twice doesn't read too nicely imho... 

What would you think of: 

> To compare that commit and _the_ previous one we can do `git diff <commit>^!`, where the `^!` tells git to compare _the commit to the one that came before it._

Reply:
Rewritten as you suggested, sounds nicer indeed.